wren-ai-service/src/providers/llm/openai.py

```diff
@@ -63,6 +65,7 @@ def get_generator(
     ):
         return CustomOpenAIGenerator(
```

Comment:
I suppose you need to add a function like this, otherwise generation_kwargs make break other LLM?

```python
    def get_generator(
        self,
        model_kwargs: Optional[Dict[str, Any]] = GENERATION_MODEL_KWARGS,
        system_prompt: Optional[str] = None,
    ):
        def _get_generation_kwargs(
            model_kwargs: Optional[Dict[str, Any]] = GENERATION_MODEL_KWARGS,
            base_url: str = OPENAI_API_BASE,
        ):
            if base_url == OPENAI_API_BASE:
                return model_kwargs
            elif model_kwargs != GENERATION_MODEL_KWARGS:
                return model_kwargs
            return None

        return CustomOpenAIGenerator(
            api_key=self._api_key,
            model=self._generation_model,
            system_prompt=system_prompt,
            generation_kwargs=_get_generation_kwargs(model_kwargs, self._base_url),
        )
```